code/game/machinery/doors/brigdoors.dm

```diff
@@ -63,6 +63,24 @@
 			stat |= BROKEN
 		update_icon()
 
+/obj/machinery/door_timer/proc/print_report()
+	var/logname = input(usr, "Name of the guilty?","[id] log name")
+	var/logcharges = input(usr, "What are they being charged with?","[id] log charges")
+
+	for(var/obj/machinery/computer/prisoner/C in prisoncomputer_list)
+
+		var/obj/item/weapon/paper/P = new /obj/item/weapon/paper(C.loc)
+		P.name = "[id] log - [logname]"
+		P.info =  "<center><b>[id] - Brig record</b></center><br><hr><br>"
+		P.info += "<center>NSS Cyberaid - Security Department</center><br>"

```

Comment:
This should populate the station name automatically using `[station_name()]` as people are maturing their designs for other stations.